src/caches/low_storage_rk_caches.jl

```diff
@@ -1371,3 +1371,71 @@ end
 function alg_cache(alg::ParsaniKetchesonDeconinck3S205,u,rate_prototype,uEltypeNoUnits,uBottomEltypeNoUnits,tTypeNoUnits,uprev,uprev2,f,t,dt,reltol,p,calck,::Type{Val{false}})
   ParsaniKetchesonDeconinck3S205ConstantCache(real(uBottomEltypeNoUnits),real(tTypeNoUnits))
 end
+
+
+
+# 2R+ low storage methods introduced by van der Hauwen
+@cache struct LowStorageRK2RPCache{uType,rateType,uNoUnitsType,TabType} <: OrdinaryDiffEqMutableCache
+  u::uType
+  k::rateType
+  Xᵢ::rateType
```

Comment:
this name is causing the resize test to fail, because `@cache` is looking for the standard names IIRC.

Reply:
Why is `Xᵢ::rateType`? In the step, it is used as `uType`, e.g. in `Xᵢ  = u + (Bᵢ[i] - A₁[i])*dt*k`.

Reply:
@ChrisRackauckas Which names? `LowStorageRK2RPCache`?

Reply:
> Why is `Xᵢ::rateType`? In the step, it is used as `uType`, e.g. in `Xᵢ = u + (Bᵢ[i] - A₁[i])*dt*k`.

Oh I previously used it as rateType. I'll change it.

Reply:
@ChrisRackauckas I changed the names of `Xᵢ` to `X` and others respectively still, I'm getting the same error.